releasenotes/notes/fix-attribute-error-bug-google-adk-8970290a34e21692.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    google-adk: Fixes an ``AttributeError`` that could occur when tracing Google ADK agent runs, due to the agent ``model`` attribute not being defined for 
```

Comment:
nit: should there also be a note about what doesn't get instrumented: https://ddtrace.readthedocs.io/en/stable/integrations.html?highlight=google%20adk#module-ddtrace.contrib.internal.google_adk?